config/ola.m4

```diff
@@ -100,9 +100,30 @@ AM_CONDITIONAL(BUILD_OLA_PROTOC_PLUGIN, test "${with_ola_protoc_plugin}" == "no"
 ])
 
 
-# PLUGIN_SUPPORT(plugin, conditional, prerequisites_found)
-# Build the specified plugin, unless it was disabled at configure time.
-# -----------------------------------------------------------------------------
+# PLUGIN_SUPPORT(plugin_key, conditional, prerequisites_found)
+#
+# Build the specified plugin if requested and its dependencies are met.
+#
+# Each plugin specified with PLUGIN_SUPPORT gets its own configure switch named
+# --enable_${plugin_key}. The three acceptable values the user can give are:
+#
+#  - auto (default): Build the plugin if its dependencies are met, otherwise
```

Comment:
That's what it is. If the user doesn't specify anything, the `enable_${plugin_key}` variable will hold the value "auto". See the call to AC_ARG_ENABLE. My sentence is not clear, I'll change it to:

    ... the three acceptable states are:
    
    - auto: Build the plugin if its dependencies are not met, otherwise
      don't. This is the default state used when the user did not specify
      anything regarding this plugin.
    ....